extensions/sql-database-projects/src/dialogs/addDatabaseReferenceDialog.ts

```diff
@@ -188,6 +188,17 @@ export class AddDatabaseReferenceDialog {
 		this.dispose();
 	}
 
+	/**
+	 * Returns undefined for settings that are an empty string, meaning they are unset
+	 * @param settingValue
+	 */
+	private validateUnsetStringSetting(settingValue?: string): string | undefined {
```

Comment:
I couldn't think of a good name for this, so open to any suggestions :)

Reply:
That's a bit of a mouthful, haha.  Maybe something more descriptive of the function rather than the use, like `ensureSetOrUndefined()`.

Reply:
sounds much better!